docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,253 @@
+
+```
+videoOverlay.getElement().pause();
+```
+
+This allows us to build custom interfaces. For example, we can build a small subclass of `L.Control` to play/pause this video overlay once it's loaded:
+
+```
+videoOverlay.on('load', function () {
+	var MyPauseControl = L.Control.extend({
+		onAdd: function() {
+			var button = L.DomUtil.create('button');
+			button.title = 'Pause';
+			button.innerHTML = '<span aria-hidden="true">⏸</span>';
+			L.DomEvent.on(button, 'click', function () {
+				videoOverlay.getElement().pause();
+			});
+			return button;
+		}
+	});
+	var MyPlayControl = L.Control.extend({
+		onAdd: function() {
+			var button = L.DomUtil.create('button');
+			button.title = 'Play';
+			button.innerHTML = '<span aria-hidden="true">▶️</span>';
+			L.DomEvent.on(button, 'click', function () {
+				videoOverlay.getElement().play();
+			});
+			return button;
+		}
+	});
+
+	var pauseControl = (new MyPauseControl()).addTo(map);
+	var playControl = (new MyPlayControl()).addTo(map);
+});
+```
+
+{% include frame.html url="example-video.html" %}
+
+### `SVGOverlay`
+
+`L.SVGOverlay` is used to load, display and provide DOM access to an SVG file over specific bounds of the map. 
+
+To add an SVG overlay [`L.SVGOverlay`](/reference.html#svgoverlay) use this:
+
+```
+var svgOverlay = L.svgOverlay(SVGElement, svgElementBounds, options);
+```
+
+It instantiates an image overlay object given an SVG element and the geographical bounds it is tied to. A viewBox attribute is required on the SVG element to zoom in and out properly.
+
+#### Creating an SVG element
+
+Let's create an SVG element:
+
+```
+var svgElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+svgElement.setAttribute('xmlns', 'http://www.w3.org/2000/svg');
+svgElement.setAttribute('viewBox', '0 0 200 200');
+svgElement.innerHTML = '<rect width="200" height="200"/><rect x="75" y="23" width="50" height="50" style="fill:red"/><rect x="75" y="123" width="50" height="50" style="fill:#0013ff"/>';
+```
+
+Alternatively, you can create the SVG element in HTML code:
+
+```
+<svg id="svg" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200"><rect width="200" height="200"/><rect x="75" y="23" width="50" height="50" style="fill:red"/><rect x="75" y="123" width="50" height="50" style="fill:#0013ff"/></svg>
+```
+
+And choose this SVG element by using a querySelector:
+
+```
+var svgElement = document.querySelector('#svg');
+```
+
+#### Creating the SVG Overlay
+
+Create the SVGOverlay with desired options similarly to ImageOverlay and VideoOverlay:
+
+```
+var latLngBounds = L.latLngBounds([[32, -130], [13, -100]]);
+map.fitBounds(latLngBounds);
+
+var svgOverlay = L.svgOverlay(svgElement, latLngBounds, {
+	opacity: 0.7,
+	interactive: true
+}).addTo(map);
+```
+
+Although SVGOverlay does not have its own unique options, it inherits a variety of options from ImageOverlay, Interactive layer and Layer.
```

Comment:
Should there be an "Options" heading (h4) before this corresponding section for each overlay, to help structure it a bit more and make it overall more readable? @Falke-Design WDYT?

Reply:
@Malvoz I think if someone needs the options he can go to the doku (over one of the 1000 links on this page 😄)
But I have no strong opinion on that, so tell us what you would like to have and then we can finally merge this PR.

Reply:
It's not about finding documentation, it's about structuring this tutorial so that it's easily skimmable. But it can otherwise be improved later if necessary (related https://github.com/Leaflet/Leaflet/issues/7254).